README.md

```diff
@@ -56,7 +56,19 @@ list of prefixes:
 
     godepgraph -p github.com,launchpad.net bitbucket.org/foo/bar
 
-
+### As part of larger project (e.g. Docker):
+
+```
+mkdir -p go2/src/github.com/docker
```

Comment:
I think that this section contains too many commands unrelated to godepgraph. Can you come up with a simpler example that primarily illustrates the usage of godepgraph?